lib/compiler/README.md

```diff
@@ -12,7 +12,15 @@ npm install @astrojs/compiler
 
 ## Usage
 
-_Note: API will change before 1.0! Use at your own discretion._
+_Note: Public APIs are likely to change before 1.0! Use at your own discretion._
+
+#### Transform `.astro` to valid TypeScript
+
+The Astro compiler can convert `.astro` syntax to a JavaScript Module whose default export generates HTML.
```

Comment:
```suggestion
The Astro compiler can convert `.astro` syntax to a TypeScript Module whose default export generates HTML.
```

Per the note below that reads "_The output code may need an additional post-processing step to generate valid **JavaScript**._"